### ISSUE TITLE
[SSE-Reasoner] Change constraint variables

### ISSUE THREAD

**Issue by eichelbe** [SSEHUB/EASyProducer#68]
... does not work

**Comment by Elscha** [comment]
Constraint variables should be overwriteable. Currently only their default values are evaluated (not the currently assigned value). The current value should behave comparable to values of other decisions.
For instance:

```
  compound CP1 {
    Integer no;
    Constraint cons = no > 0;
  }

  compound CP2 refines CP1 {
    cons = null;
  }

  CP2 cmp = {no = 0};
```

Should not detect an error.

**Comment by eichelbe** [comment]
Supported in new version of reasoner, fixed before, tested in 1473c34cfc9467ee3657e714d7b0937e212231d0
